cli/tests/pcluster/config/test_json_param_types.py

```diff
@@ -94,26 +94,16 @@ def boto3_stubber_path():
 
 
 @pytest.mark.parametrize(
-    "queues, expected_warnings",
+    "queues",
     [
-        (
-            ["queue1"],
-            "WARNING: EFA was enabled on queue 'queue1', but instance type 'c4.xlarge' does not support EFA.\n"
-            "WARNING: EFA was enabled on queue 'queue1', but instance type 't2.xlarge' does not support EFA.\n"
-            "WARNING: EFA was enabled on queue 'queue1', but instance type 'm6g.xlarge' does not support EFA.\n",
-        ),
-        (["queue2"], ""),
-        (
-            ["queue1", "queue2"],
-            "WARNING: EFA was enabled on queue 'queue1', but instance type 'c4.xlarge' does not support EFA.\n"
-            "WARNING: EFA was enabled on queue 'queue1', but instance type 't2.xlarge' does not support EFA.\n"
-            "WARNING: EFA was enabled on queue 'queue1', but instance type 'm6g.xlarge' does not support EFA.\n",
-        ),
-        ([], ""),
-        (["queue3"], ""),
+        (["queue1"]),
+        (["queue2"]),
+        (["queue1", "queue2"]),
+        ([]),
+        (["queue3"]),
     ],
```

Comment:
Given the fact we moved these validation steps into the `test_queue_validator` probably it's now useless to have so many tests for test_config_to_json that are not triggering any validation path.

Reply:
There is still the validation "Check that created json params match the expected ones"